src/ontology/mondo-edit.obo

```diff
@@ -422565,6 +422587,86 @@ is_a: MONDO:0002558 {source="Orphanet:590539"} ! melanotic neurilemmoma
 is_a: MONDO:0016752 {source="Orphanet:590539"} ! benign peripheral nerve sheath tumor
 is_a: MONDO:0018078 {source="Orphanet:590539"} ! soft tissue sarcoma
 
+[Term]
+id: MONDO:0035349
+name: localized dystrophic epidermolysis bullosa
+def: "A localized form of dystrophic epidermolysis bullosa characterized by blisters confined primarily to the hands and feet (acral form) or to the pretibial region (pretibial form). Nail dystrophy or loss is common and may be an isolated finding (nail only form). This disease can be inherited via autosomal dominant or autosomal recessive inheritance." [Orphanet:595356]
+subset: ordo_disease
+synonym: "localized DEB" EXACT [Orphanet:595356]
+xref: Orphanet:595356 {source="MONDO:equivalentTo"}
+is_a: MONDO:0017608 {source="Orphanet:595356"} ! dystrophic epidermolysis bullosa
+
+[Term]
+id: MONDO:0035354
+name: IgG4-related systemic disease
+subset: ordo_disease
+synonym: "IgG4-RD" BROAD ABBREVIATION [Orphanet:596448]
+synonym: "IgG4-related disease" BROAD [Orphanet:596448]
+xref: Orphanet:596448 {source="MONDO:equivalentTo"}
+is_a: MONDO:0017287 {source="Orphanet:596448"} ! IgG4-related disease
+
+[Term]
+id: MONDO:0035357
+name: portosinusoidal vascular disease
+subset: ordo_disease
+synonym: "PSVD" EXACT ABBREVIATION [Orphanet:596937]
+xref: ICD10:K76.8 {source="Orphanet:596937/ntbt", source="Orphanet:596937"}
+xref: Orphanet:596937 {source="MONDO:equivalentTo"}
+is_a: MONDO:0002405 {source="Orphanet:596937"} ! hepatic vascular disease
+
+[Term]
+id: MONDO:0035362
+name: TRIM22-related inflammatory bowel disease
+subset: ordo_disease
+synonym: "TRIM22-related IBD" EXACT [Orphanet:597201]
+xref: Orphanet:597201 {source="MONDO:equivalentTo"}
+is_a: MONDO:0005265 {source="Orphanet:597201"} ! inflammatory bowel disease
+
+[Term]
+id: MONDO:0035363
+name: IRF2BPL-related regressive neurodevelopmental disorder-dystonia-seizures syndrome
+subset: ordo_disease
+xref: Orphanet:597623 {source="MONDO:equivalentTo"}
+is_a: MONDO:0002254 {source="https://orcid.org/0000-0001-5208-3432"} ! syndromic disease
+relationship: disease_has_basis_in_dysfunction_of http://identifiers.org/hgnc/14282 {source="Orphanet:597623"}
+
+[Term]
+id: MONDO:0035369
+name: MTHFS-related developmental delay-microcephaly-short stature-epilepsy syndrome
+subset: ordo_disease
+xref: Orphanet:597874 {source="MONDO:equivalentTo"}
+is_a: MONDO:0000001 {source="Orphanet:597874"} ! disease or disorder
+is_a: MONDO:0017313 {source="Orphanet:597874"} ! disorder of folate metabolism and transport
+is_a: MONDO:0019046 {source="Orphanet:597874"} ! leukodystrophy
+is_a: MONDO:0019058 {source="Orphanet:597874"} ! neurometabolic disease
+
+[Term]
+id: MONDO:0035370
+name: ALPI-related inflammatory bowel disease
```

Comment:
is this mendelian? if so should be in omim

if not what is this? susceptibility? is it really a subtype of IBD?

Reply:
https://github.com/monarch-initiative/mondo/issues/3543